Core/Debugger/Breakpoints.cpp

```diff
@@ -644,12 +644,15 @@ void CBreakPoints::Update(u32 addr)
 			Core_WaitInactive(200);
 			resume = true;
 		}
-		
-		// In case this is a delay slot, clear the previous instruction too.
-		if (addr != 0)
-			MIPSComp::jit->InvalidateCacheAt(addr - 4, 8);
-		else
-			MIPSComp::jit->ClearCache();
+
+		std::lock_guard<std::recursive_mutex> guard(MIPSComp::jitLock);
```

Comment:
Technically, this could be unlocked before Core_EnableStepping by adding another scope here. Not sure if it matters.

Reply:
Changed.  Sometimes I try too hard to avoid constructless scopes (I find them harder to read.)

-[Unknown]